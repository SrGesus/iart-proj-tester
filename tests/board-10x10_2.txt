VB	LH	LV	VD	FE	LV	VD	VB	LV	FD
LH	FE	FB	VB	FC	FB	LH	BD	LV	VD
LH	LV	FC	FB	VD	BE	BE	VE	FD	LH
LV	BC	BC	VE	VB	VB	VC	VD	VE	LH
LH	LH	FE	VC	VC	VE	VD	LH	VC	VE
BB	VD	FB	BD	VD	VC	BC	BE	VC	FD
LH	FC	VB	VE	BD	FE	LV	FC	VC	BC
VE	BE	BB	LH	VD	FC	VC	BB	VB	LH
FB	BD	BC	FD	FD	BE	BD	LV	FD	LV
VB	VC	VB	BB	BB	VB	VC	FB	FC	VB
