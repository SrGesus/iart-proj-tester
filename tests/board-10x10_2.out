VB	LH	LH	VE	FD	LH	VE	VB	LH	FE
LV	FB	FD	VC	FB	FB	LV	BD	LH	VE
LV	LV	FB	FB	VD	BE	BD	VC	FB	LV
LV	BD	BE	VD	VE	VD	VC	VB	VC	LV
LV	LV	FC	VB	VC	VB	VE	LV	VB	VC
BD	VC	FD	BE	VB	VC	BD	BC	VC	FB
LV	FB	VB	VC	BD	FE	LV	FB	VB	BE
VD	BE	BD	LH	VC	FB	VD	BC	VC	LV
FB	BD	BE	FB	FB	BD	BB	LH	FE	LV
VD	VC	VD	BC	BC	VC	VD	FE	FD	VC
